canonical-web-and-design/ubuntu.com
# Certified category copy

## Done

- Copy updates and setup for certified/<category> views
- Refactor category view templates into one single template
- Removed `base_certification` include template, as this doesn't provide any value (only a few linesand we need also direct access to `meta_copydoc` in `base.html`)

## QA

- Navigate to https://ubuntu-com-10207.demos.haus
- In the top navigation try each link (Desktops, Laptops, Devices etc...)
- Make sure the following work as on https://ubuntu.com/certified:
  - Filters, select multiple filters and apply them
  - Clear filters
  - Click to show or hide filters when applicable
  - Results per page work as expected and they do not duplicate the `limit` query
  - Pagination works as expected
- Check copy docs are correctly linked and the content matches. To see this, inspect in developer tools and look for the meta tag `copy_doc` if you don't have the copy doc Chrome plugin installed.

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4290
